WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -312,6 +316,18 @@ final class DashboardViewModel {
     }
 }
 
+// MARK: Theme install
+//
+private extension DashboardViewModel {
+    /// Installs themes for newly created store.
+    ///
+    func installPendingTheme() {
+        Task { @MainActor in
+            try? await themeInstaller.installPendingTheme(siteID: siteID)
+        }
+    }
```

Comment:
Nit: should we rename the methods to `installPendingThemeIfNeeded` to clarify the intention? Also, let's add a `DDLogError` if the installation fails.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11478/commits/9bebfbe06924a6ec3a53a3b61af6ceed52170720 and 7ce4868e963087a1ca07f05b86e2cd12fa5d5bd5